src/main/java/com/socrata/api/SodaDdl.java

```diff
@@ -355,16 +352,9 @@ public ClientResponse issueRequest() throws LongRunningQueryException, SodaError
 
         try {
             final ClientResponse response = requester.issueRequest();
-            return mapper.readValue(response.getEntity(InputStream.class), AssetResponse.class);
-            //return response.getEntity(AssetResponse.class);
+            return response.getEntity(AssetResponse.class);
```

Comment:
response.getEntity doesn't work with creating assets /api/assets.  Keep the original mapper.readValue and error handlers.